### DISCUSSION TITLE
Route user to a new url from a server component

### DISCUSSION THREAD

**Discussion by JoeRoddy** [vercel/next.js#71987] [App Router] [answered]
Is nextNavigation.redirect() the only API for this?

Is there any other way to navigate a user to a new page, via server components only, without throwing an error?

**Comment by icyJoseph** [comment, marked as answer]
For now yeah... unless you want to introduce middleware for the route segment (using a matcher), but that's not always an option I gues.s

**Comment by JoeRoddy** [comment]
Thanks!
